fedimint-core/src/api.rs

```diff
@@ -772,6 +772,21 @@ impl InviteCode {
         ])
     }
 
+    pub fn new_max_guardians_code(
+        peer_to_url_map: BTreeMap<PeerId, SafeUrl>,
```

Comment:
Maybe we can make this a reference? If we don't consume all entries we don't need to clone all of them, but could rather call `clone` inside the `map` statement.